### PR TITLE
Move Reset Kubernetes button to Troubleshooting

### LINKED ISSUE #2553: Unable to `nerdctl push container` after `nerdctl login` when using `pass` credsStore on Linux
### Actual Behavior

I'm unable to push a container image by using `nerdctl push thehejik/busybox:latest` after deleting default docker's `~/.docker/config.json` and providing my personal dockerhub credentials. Docker/moby RD backend works normally.

Rancher-desktop version 1.4.1-373-g9051f530 built from main branch installed on openSUSE 15.3 and Ubuntu 20.04.

### Steps to Reproduce

* Install RD
* delete ~/.docker/config.json (if exists)
* run RD and initialize with default values
* setup `pass` according to https://docs.rancherdesktop.io/getting-started/installation#pass-setup:
```
$ gpg --generate-key # provide full name, e-mail and confirm, do not provide a passphrase
> 2E1C8E4D37F7D82EF02176BC5B300A52EEE63CAB
$ pass init 2E1C8E4D37F7D82EF02176BC5B300A52EEE63CAB
mkdir: created directory '/home/thehejik/.password-store/'
Password store initialized for 2E1C8E4D37F7D82EF02176BC5B300A52EEE63CAB
$ pass list
Password Store
└── docker-credential-helpers
    └── aHR0cHM6Ly9pbmRleC5kb2NrZXIuaW8vdjEv
        └── thehejik
```
* run `nerdctl login` and provide your private DockerHub credentials (wait for `Login Succeeded`)
* pull some image eg by. `nerdctl pull busybox`
* re-tag the pulled image by `nerdctl <id_of_busybox_image> thehejik/busybox:latest` (replace `thehejik` by your docker username)
* try to push the retagged image `nerdctl push thehejik/busybox:latest`

### Result

The busybox image is not pushed with Error message `FATA[0002] server message: insufficient_scope: authorization failed`:
```bash
$ nerdctl push thehejik/busybox:latest
INFO[0000] pushing as a reduced-platform image (application/vnd.docker.distribution.manifest.list.v2+json, sha256:c12b4701670111b6d3518ab989138dbf9b284d464982ceaa21128e280242e0f9) 
index-sha256:c12b4701670111b6d3518ab989138dbf9b284d464982ceaa21128e280242e0f9:    waiting        |--------------------------------------| 
manifest-sha256:dcdf379c574e1773d703f0c0d56d67594e7a91d6b84d11ff46799f60fb081c52: waiting        |--------------------------------------| 
config-sha256:62aedd01bd8520c43d06b09f7a0f67ba9720bdc04631a8242c65ea995f3ecac8:   waiting        |--------------------------------------| 
elapsed: 1.8 s                                                                    total:   0.0 B (0.0 B/s)                                         
FATA[0002] server message: insufficient_scope: authorization failed
```

### Expected Behavior

The image should be pushed to DockerHub as with Docker.

### Additional Information

Works ok with docker/moby RD backend. Also docker installed on the system is working fine.

### Rancher Desktop Version

1.4.1-main

### Rancher Desktop K8s Version

any

### Which container runtime are you using?

containerd (nerdctl)

### What operating system are you using?

Ubuntu

### Operating System / Build Version

Ubuntu 20.04, openSUSE 15.3

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

deb

### Windows User Only

_No response_

### PR DESCRIPTION
**Pull request by rak-phillip**
This moves the Reset Kubernetes button over to the Troubleshooting page in preparation for the new preferences. 

Duplicated buttons will be removed in #2524

closes #2553

### CODE REVIEW

File: src/assets/translations/en-us.yaml [hunk 1]
```diff
@@ -4574,8 +4574,14 @@ troubleshooting:
     title: Kubernetes
     resetKubernetes:
       title: Reset Kubernetes
-      description: Resetting Kubernetes will delete workloads and configuration.
+      description:  Resetting Kubernetes will delete all workloads and configuration.
```
Comment (ericpromislow) [inline src/assets/translations/en-us.yaml:4577]: The linter doesn't care, but please remove the extra space before "Resetting".

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Everything else works fine and looks good. Just the trivial extra space.

File: src/assets/translations/en-us.yaml (hunk 1 above)
Comment (rak-phillip) [reply src/assets/translations/en-us.yaml]: Good catch, thanks!

**Comment by jandubois** [comment]
> Duplicated buttons will be removed in #2524

That issue is about not making the settings modal, and removing the parent; so is this the right reference?
 
> closes #2553

That issue does not exist, afaict.

**Review by ericpromislow — APPROVED** [review]
Looks fine

**Pull request merged**